Objects/abstract.c

```diff
@@ -1732,7 +1732,14 @@ PySequence_SetItem(PyObject *s, Py_ssize_t i, PyObject *o)
                 i += l;
             }
         }
-        return m->sq_ass_item(s, i, o);
+        if (m->sq_ass_item)
+            return m->sq_ass_item(s, i, o);
+        else if (m->sq_ass_item){
```

Comment:
Did I miss something or is this check useless?

Reply:
Oops. You are right!